.github/workflows/build-test.yml

```diff
@@ -146,12 +146,12 @@ jobs:
     - if: contains( matrix.os, 'ubuntu-latest') && steps.changes.outputs.DO_BUILD
       name: Do Linux tests
       run: |
-        cd ../tests && qmake CONFIG+=debug CONFIG+=DEV_MODE DEFINES+=INTEGRATION_TESTS && make -s -j$(nproc)
+        cd ../tests && qmake CONFIG+=debug CONFIG+=DEV_MODE DEFINES+=INTEGRATION_TESTS && make -s -j 8
```

Comment:
Why 8? And why not `$(nproc)`?

Reply:
nproc doesn't work on windows so had to fallback.